pq-crypto/kyber_r2/fips202_kyber_r2.c

```diff
@@ -120,6 +114,13 @@ static void KeccakF1600_StatePermute(uint64_t *state) {
 	Asu = state[24];
 
 	for (round = 0; round < NROUNDS; round += 2) {
+        uint64_t Da, De, Di, Do, Du;
+        uint64_t Eba, Ebe, Ebi, Ebo, Ebu;
+        uint64_t Ega, Ege, Egi, Ego, Egu;
+        uint64_t Eka, Eke, Eki, Eko, Eku;
+        uint64_t Ema, Eme, Emi, Emo, Emu;
+        uint64_t Esa, Ese, Esi, Eso, Esu;
```

Comment:
Indent doesn't match?

Reply:
All the indent mismatches should be fixed now.